src/scripts/main.js

```diff
@@ -1,3 +1,41 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+logo.addEventListener('click', () => {
+  handleClick();
```

Comment:
It's unnecessary to wrap the `handleClick` function in another anonymous function. You can pass the `handleClick` function directly as the event handler. So, you can simplify this line to `logo.addEventListener('click', handleClick);`